server/src/test/java/org/elasticsearch/cluster/metadata/AliasMetaDataTests.java

```diff
@@ -54,6 +55,21 @@ public void testSerialization() throws IOException {
         assertThat(after, equalTo(before));
     }
 
+    @Override
+    protected void assertEqualInstances(AliasMetaData expectedInstance, AliasMetaData newInstance) {
+        assertNotSame(newInstance, expectedInstance);
+        if (expectedInstance.writeIndex() == null) {
+            expectedInstance = AliasMetaData.builder(expectedInstance.alias())
+                .filter(expectedInstance.filter())
+                .indexRouting(expectedInstance.indexRouting())
+                .searchRouting(expectedInstance.searchRouting())
+                .writeIndex(Boolean.FALSE)
```

Comment:
why is that force false?

Reply:
just leaving the static nature of this test, will update